.gitignore

```diff
@@ -20,3 +21,4 @@ deb
 /proto/scripts/protoc-gen-gosyncthing
 /gui/next-gen-gui
 .idea
+.vscode
```

Comment:
Let's not add this; the .idea one is a historical mistake, and we shouldn't have to track which editors exist

Reply:
On other hand it is making life or developers easy. I don't think there are that many editors widely used. But if you wish...

Reply:
There is still the possibility to add local ignore patterns in Git's config. Suitable for every programme's own taste and tooling. Thus I agree we should keep all those dort-directories out of the committed ignore list.